tests/testAutopas/CMakeLists.txt

```
@@ -5,7 +5,7 @@ file(
     "*.h"
 )
 
-add_executable(runTests ${MY_SRC})
+add_executable(runTests ${MY_SRC} tests/forEach/ContainerReduceTest.cpp tests/forEach/ContainerReduceTest.h)
```

```suggestion
add_executable(runTests ${MY_SRC})
```
Should be covered by glob?